src/common

```diff
@@ -35,6 +35,9 @@ endif
   CFLAGS ?= -Wall -O2
   CP_CFLAGS = -MMD -MP
   CXXFLAGS ?= -Wall -O2
+  CXXFLAGS += -flto
+  LINKFLAGS += -flto
+  AR = gcc-ar
```

Comment:
I guess CI will probably tell us this, but will this work with clang builds?

Reply:
Is this feature supposed to be stable in g++-5 ?

Reply:
afaict, yes.